src/sql/workbench/browser/modelComponents/media/card.css

```diff
@@ -122,7 +113,7 @@
 	border-radius: 50%;
 	background-color: white;
 	border-width: 1px;
-	border-color: rgb(0, 120, 215);
+	border-color: rgb(214, 214, 214);
```

Comment:
was this a contrast issue for accessibility purposes?

Reply:
the old color is blueish, it might confuse people since that is normally the focused border color. I am changing it to the same as the border color when the card is not being focused.

Reply:
And you verified this passes accessibility requirements for contrast in both light and dark mode?